src/main/scala/org/broadinstitute/dsde/workbench/avram/api/AvramRoutes.scala

```diff
@@ -1,9 +1,21 @@
 package org.broadinstitute.dsde.workbench.avram.api
 
+import java.sql.{Connection, DriverManager, PreparedStatement, ResultSet}
+
 import com.google.api.server.spi.config.{Api, ApiMethod}
-import org.broadinstitute.dsde.workbench.avram.util.Logger
+import com.typesafe.config.ConfigFactory
+import org.broadinstitute.dsde.workbench.avram.util.{DataSource, Logger}
+
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, Future}
+import scala.concurrent.ExecutionContext.Implicits.global
+import slick.jdbc.PostgresProfile.api._
+
+import scala.beans.BeanProperty
 
 case class Pong()
+case class Now(@BeanProperty message: String)
```

Comment:
I agree with what Anu said in person that it's annoying to have to annotate all properties with `@BeanProperty`. It's unclear to me what exactly is insisting that these objects have getters/setters.